Sources/FacebookCoreTests/GraphRequestQueryItemBuilderTests.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+//
+// You are hereby granted a non-exclusive, worldwide, royalty-free license to use,
+// copy, modify, and distribute this software in source code or binary form for use
+// in connection with the web services and APIs provided by Facebook.
+//
+// As with any software that integrates with the Facebook platform, your use of
+// this software is subject to the Facebook Developer Principles and Policies
+// [http://developers.facebook.com/policy/]. This copyright notice shall be
+// included in all copies or substantial portions of the software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// swiftlint:disable explicit_type_interface multiline_arguments literal_expression_end_indentation force_unwrapping
+
+@testable import FacebookCore
+import XCTest
+
+class GraphRequestQueryItemBuilderTests: XCTestCase {
+
+  var dictionary = [String: AnyHashable]()
```

Comment:
Why class state and not function state?

Reply:
This allows me to only cast as [String: AnyHashable] in one place. Otherwise it infers it as [String: Any] every time I declare and I have to cast it back. :/